extensions/cornerstone-dicom-sr/src/commandsModule.ts

```diff
@@ -61,6 +82,13 @@ const commandsModule = (props: withAppTypes) => {
       window.location.assign(objectUrl);
     },
 
+    /**
+     * Download the CSV report for the measurements.
+     */
+    downloadCSVMeasurementsReport: ({ measurementFilter }) => {
```

Comment:
since this PR is going through another iteration, can you move this to also cornerstone commands too?